tests/test_helper.bash

```diff
@@ -1,4 +1,254 @@
+    -H 'content-type: application/json' \
+    -d '
+        {
+          "license_title": "License not specified",
+          "maintainer": null,
+          "relationships_as_object": [],
+          "private": true,
+          "maintainer_email": null,
+          "num_tags": 1,
+          "metadata_created": "2019-12-18T19:01:33.429530",
+          "metadata_modified": "2019-12-18T19:02:54.841495",
+          "author": null,
+          "author_email": null,
+          "state": "active",
+          "version": null,
+          "type": "dataset",
+          "resources": [
+            {
+              "conformsTo": "",
+              "cache_last_updated": null,
+              "describedByType": "",
+              "labels": {
+                "accessURL new": "Access URL",
+                "conformsTo": "Conforms To",
+                "describedBy": "Described By",
+                "describedByType": "Described By Type",
+                "format": "Media Type",
+                "formatReadable": "Format",
+                "created": "Created"
+              },
+              "webstore_last_updated": null,
+              "clear_upload": "",
+              "state": "active",
+              "size": null,
+              "describedBy": "",
+              "hash": "",
+              "description": "",
+              "format": "CSV",
+              "mimetype_inner": null,
+              "url_type": null,
+              "formatReadable": "",
+              "mimetype": null,
+              "cache_url": null,
+              "name": "Test Resource",
+              "created": "2019-12-18T19:02:54.448285",
+              "url": "https://www.bia.gov/tribal-leaders-csv",
+              "upload": "",
+              "webstore_url": null,
+              "last_modified": null,
+              "position": 0,
+              "resource_type": "file"
+            }
+          ],
+          "num_resources": 1,
+          "tags": [
+            {
+              "vocabulary_id": null,
+              "state": "active",
+              "display_name": "test",
+              "id": "65c76784-e271-4eb1-9778-a738622a1a3d",
+              "name": "test"
+            }
+          ],
+          "tag_string": "test",
+          "groups": [],
+          "license_id": "notspecified",
+          "relationships_as_subject": [],
+          "organization": "test-organization-'$RNDCODE'",
+          "isopen": false,
+          "url": null,
+          "notes": "The description of the test dataset",
+          "owner_org": "test-organization-'$RNDCODE'",
+          "bureau_code": "010:00",
+          "contact_email": "tester@fake.com",
+          "contact_name": "Tester",
+          "modified": "2019-12-18",
+          "public_access_level": "public",
+          "publisher": "Department of the Interior",
+          "unique_id": "doi-'$RNDCODE'",
+          "title": "Test Dataset '$RNDCODE'",
+          "name": "test-dataset-'$RNDCODE'",
+          "program_code": "010:001"
+        }'
```

Comment:
Maybe move this JSON payload to a test data fixture?

Reply:
I moved JSON data to external files [here](https://github.com/GSA/catalog.data.gov/pull/10/commits/4ef22440d2e556bc67ef2f477467e8c2dda8db30)